client/App.tsx

```diff
@@ -42,9 +69,80 @@ export default function HomePage() {
                 </button>
             </div>
             <div className="container">
-                <section id="pageOne" />
+                <section id="pageOne">
```

Comment:
Sould each `<section>`be broken up into it's own react component for easier website updating in the future?

Reply:
I think the sections won't get too complicated, so we are good for now. If they do, we can brake them.